test/perf/PerfConv.cpp

```diff
@@ -16,18 +16,12 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include <cassert>
-#include <iostream>
-#include <string>
```

Comment:
Question: is it not good form to include things that are needed in a file, even if they may be included implicitly in other header files? Or is it preferred to aim for conciseness.

Reply:
This is arguable. Some argue that is better to include all headers that are required to compile the `.cpp` file, and avoid relying on transitive inclusion.  In this case <string> is already included in `ModelLib.hpp` which is the header file that is most closely associated with this `.cpp` file so I went for conciseness. And the <iostream> header is not required for this .cpp file.